sreejinsreenivasan/linkdot
# fix: added back button

Feature: Add search result back button 
QA Notes:
---
Do a search
and in the search result page should have back button containing search query

1. Make sure back button leads to `explore` page
2. Make sure the query display test is correct
